README.md

```diff
@@ -1 +1,62 @@
 # Rbutils
+
+# Description
+
+Upfluence-utils is a utility box for ruby ​​projects. Some important abstractions are:
+
+- Sentry error logger
+
+- Base Api Endpoint
+
+- Middlewares
+
+- Mixins
+
+## Usage
+
+Add in your gemfile the `gem 'upfluence-utils'` and run `bundle install`
```

Comment:
I always found it very confusing that the repo name was different than the gem name.
It makes it harder to find it if you don't know it (not sure we can do much about it though)

Reply:
Definitely a good idea

Reply:
changed, some link should be broken because we need to change the repo name, but should be good after that